client-v2/src/shared/types/Collection.ts

```diff
@@ -7,6 +7,7 @@ interface Group {
   articleFragments: string[];
 }
 
+type ProperStages = 'draft' | 'live';
```

Comment:
Suggestions from the team - CurrentStages? ActiveStages?

Or, we could name ProperStages as Stages and rename Stages to CollectionItemSets or similar?